### PR TITLE
Add Menu Logic

### LINKED ISSUE #127: Add logic to each menu
- [x] Conditions
- [x] Positions

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #127 

There's room for improvement.
- Wrap and unwrap only log something.
- Redeem could be visible only for address with enough balance. But it's expensive to do it for each row.
- Maybe we can avoid using params for this because when you change a position/condition should update the URL, I don't really like that. Opinions?

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr144--conditionaltokensexplorer.review.gnosisdev.com

**Comment by lmcorbalan** [comment]
@hpmaxi I do agree with you about using route params in Condition/Position contexts. I can imagine 2 options (although neither of them totally convince me):
1- using localStorage for saving condition/position id, then reeding and removing it from there in the context provider :see_no_evil:
2- similar to 1 but using a new context provider :scream_cat:

**Comment by hpmaxi** [comment]
> I don't have a strong opinion about what route to take: show it on both, or don't, but keep it coherent.
I'll go for showing it on both.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr144--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr144--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr144--conditionaltokensexplorer.review.gnosisdev.com

File: src/pages/Routes/index.tsx [hunk 1]
```diff
@@ -38,10 +38,10 @@ export const Routes: React.FC = () => {
...
       <ProtectedRoute component={PositionDetails} exact path="/positions/:positionId" />
       <ProtectedRoute component={PrepareCondition} path="/prepare" />
-      <ProtectedRoute component={SplitPosition} path="/split" />
-      <ProtectedRoute component={ReportPayoutsContainer} path="/report" />
-      <ProtectedRoute component={RedeemPosition} path="/redeem" />
-      <ProtectedRoute component={MergePositions} path="/merge" />
+      <ProtectedRoute component={SplitPosition} path="/split/:conditionId?" />
+      <ProtectedRoute component={ReportPayoutsContainer} path="/report/:conditionId?" />
+      <ProtectedRoute component={RedeemPosition} path="/redeem/:positionId?" />
+      <ProtectedRoute component={MergePositions} path="/merge/:conditionId?" />
```
Comment (lmcorbalan) [inline src/pages/Routes/index.tsx:44]: I think we don't need url params anymore

File: src/pages/PositionsList/index.tsx [hunk 2]
```diff
@@ -44,10 +38,56 @@ export const PositionsList = () => {
...
+
+      // if (!isConnected) {
+      //   return [detailsOption]
+      // } else {
+      //   if (userBalance.gt(0)) {
+      //     return [detailsOption, redeemOption, wrapERC20Option, unwrapOption]
+      //   } else {
+      //     return [detailsOption, wrapERC20Option, unwrapOption]
+      //   }
+      // }
```
Comment (lmcorbalan) [inline src/pages/PositionsList/index.tsx:84]: Should we remove this commented code?

File: src/pages/PositionsList/index.tsx [hunk 3]
```diff
@@ -44,10 +38,56 @@ export const PositionsList = () => {
   const { _type: status, networkConfig } = useWeb3ConnectedOrInfura()
   const { data, error, loading } = usePositions(positionIdToSearch)
   const history = useHistory()
+  const { setValue } = useLocalStorage('positionid')
+
+  //const isConnected = status === 'connected'
```
Comment (lmcorbalan) [inline src/pages/PositionsList/index.tsx:43]: Should we remove this commented code?

File: src/pages/ConditionsList/index.tsx [hunk 4]
```diff
@@ -44,12 +46,51 @@ export const ConditionsList: React.FC = () => {
...
+        },
+      }
+
+      // if (!isConnected) {
+      //   return [detailsOption]
+      // }
+
+      // return address && oracle.toLowerCase() === address.toLowerCase()
+      //   ? [detailsOption, splitOption, mergeOption, reportOption]
+      //   : [detailsOption, splitOption, mergeOption]
```
Comment (lmcorbalan) [inline src/pages/ConditionsList/index.tsx:89]: Should we remove this commented code?

File: src/pages/ConditionsList/index.tsx [hunk 5]
```diff
@@ -44,12 +46,51 @@ export const ConditionsList: React.FC = () => {
...
   const isSearching = conditionIdToSearch && loading
   const history = useHistory()
-  const dropdownItems = [
-    { text: 'Details' },
-    { text: 'Split Position' },
-    { text: 'Merge Positions' },
-    { text: 'Report Payouts' },
-  ]
+  //const { _type: status, address } = useWeb3ConnectedOrInfura()
+  //const isConnected = status === 'connected'
```
Comment (lmcorbalan) [inline src/pages/ConditionsList/index.tsx:50]: Should we remove this commented code?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr144--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**